### ISSUE TITLE
JIG table - privacy labels editing options

### ISSUE THREAD

**Issue by GaliShapira** [ji-devs/ji-cloud#1937]
Hi @mastapegs @MendyBerger @corinnewo 
I updated the privacy labels selection.
1. There is an edit icon to start editing mode
2. On editing mode, a dropdown with the tags appears.
3. Multi-select is possible, in this screen I added a table of the selection options.
Pls let me know if you have any comments.

Zeplin:
https://zpl.io/2yReLvK

![Uploading Screen Shot 2021-11-24 at 10.20.59.png…]()

**Comment by MendyBerger** [comment]
This one is invalid, either public or private has to be selected

![image](https://user-images.githubusercontent.com/12537668/143234932-4b8be1a3-1044-401c-b028-30ce2c5ef4e1.png)

**Comment by GaliShapira** [comment]
@MendyBerger 
The Block cancel any option to share this link. It cancel them both.
Its almost as delete.

**Comment by MendyBerger** [comment]
@GaliShapira but once you unblock where should it go back to? Public or private?

**Comment by MendyBerger** [comment]
I think it makes perfect sense for a Jig to be public and blocked, it just means that once it's unblocked it'll be publicly indexed, but is blocked for now

**Comment by corinnewo** [comment]
You are right, @MendyBerger - Blocking will prevent it from being found at all on the system. But once it is unblocked, it should revert to its previous status of either public or private (most probably public).
